src/index.css

```diff
@@ -1,47 +1,145 @@
-.header {
+@import url('https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap');
```

Comment:
- [ ] Nice working on your styling, kindly make sure your design is exactly the same as the one provided.
### Your Design:
![--s-design](https://user-images.githubusercontent.com/48371156/187009967-df05ab9d-a4b0-4dc1-a590-63f026fd4778.png)

### Template
![--f-design](https://user-images.githubusercontent.com/48371156/187009982-0f8e5b27-5bcc-46a8-a1b6-3c640bab0c70.png)

Reply:
done